src/Types/ContractInitializedEvent.cs

```diff
@@ -0,0 +1,51 @@
+using Concordium.Sdk.Exceptions;
+
+namespace Concordium.Sdk.Types;
+
+/// <summary>
+/// Data generated as part of initializing a single contract instance.
+/// </summary>
+public class ContractInitializedEvent
+{
+    /// <summary>
+    /// Contract version.
+    /// </summary>
+    public ContractVersion ContractVersion { get; init; }
+    /// <summary>
+    /// A reference to a smart contract module deployed on the chain.
+    /// </summary>
+    public ModuleReference ModuleReference { get; init; }
+    /// <summary>
+    /// Represents a contract address.
+    /// </summary>
+    public ContractAddress ContractAddress { get; init; }
+    /// <summary>
+    /// Represents a CCD amount.
+    /// </summary>
+    public CcdAmount Amount { get; init; }
+    /// <summary>
+    /// A contract name.
```

Comment:
Please elaborate that this will be in the format `init_*`, i.e., there will be this `init_` prefix.